fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -175,6 +177,11 @@ public enum OlapTableState {
 
     private AutoIncrementGenerator autoIncrementGenerator;
 
+    @SerializedName(value = "visibleVersion")
```

Comment:
Move these to Table.java 's `TableAttributes`, so no need to update meta version